src/index.html

```diff
@@ -11,12 +11,11 @@
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
-
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="canonical" href="http://localhost:4200/dashboard" />
```

Comment:
why are we using localhost ?

Reply:
I used it for testing purpose. I have changed it as global URL in the code now. Will push the code as soon as I am done with all the changes. Sorry for the inconvenience.
- Suyash.